index.js

```diff
@@ -75,7 +61,16 @@ export default class ClapprStats extends ContainerPlugin {
 
   stopReporting() {
     this._buildReport()
+
     clearInterval(this._intervalId)
+    this._newMetrics()
+
+    this.container.off(Events.CONTAINER_TIMEUPDATE, this.onContainerUpdateWhilePlaying)
```

Comment:
@towerz do you agree? ( I mean, instead of unlistening all events (or some) we should use `stopListening` and `bindEvents`, it's kinda what we do everytime we need to reset a plugin)